pom.xml

```diff
@@ -61,6 +61,14 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
Isn't there already the same config in the uber parent?

Reply:
Könntet ihr bitte unter der entsprechenden Stelle kommentieren. Ich sehe die Anmerkung am Tag `<plugin>` und nicht das, worauf sich die Anmerkung bezieht.

Reply:
No, in Uber Parent, the setting is the following `<sourcepath>${project.build.directory}/generated-sources/delombok</sourcepath>`.

Reply:
The idea is to have this for both delombok and xjc specified once in the uber parent and nowhere else. That pom defines both JAXB code generation and Lombok as allowed frameworks for jQA and thus it makes sense to have the appropriate Javadoc config there as well.

Reply:
Ok, now I got it. I will implement it, as you suggested.

Reply:
Removed, commited and pushed.

Reply:
It’s not removed here (at least as GitHub shows it to me)

Reply:
Updated and removed.